theme/less/gouvfr/banners.less

```diff
@@ -0,0 +1,64 @@
+@import "~bootstrap/less/mixins";
+@import "variables";
+
+/**
+ * Banner styles.
+ */
+
+ .banner-gradient(@color: #003b80, @end: #0053b3) {
+  // @end: lighten(@color, 22%);
+  background-size: cover;
+  background: @color; /* Old browsers */
+  background: -moz-linear-gradient(45deg,  @color 0%, @end 100%); /* FF3.6+ */
+  background: -webkit-gradient(linear, left bottom, right top, color-stop(0%, @color), color-stop(100%, @end)); /* Chrome,Safari4+ */
+  background: -webkit-linear-gradient(45deg, @color 0%, @end 100%); /* Chrome10+,Safari5.1+ */
+  background: -o-linear-gradient(45deg, @color 0%, @end 100%); /* Opera 11.10+ */
+  background: -ms-linear-gradient(45deg, @color 0%, @end 100%); /* IE10+ */
+  background: linear-gradient(45deg, @color 0%, @end 100%); /* W3C */
+  filter:
+      e(%("progid:DXImageTransform.Microsoft.gradient(startColorstr='%d', endColorstr='%d',GradientType=0)", @color, @end));
+  }
```

Comment:
```
background-image: -webkit-gradient(linear, right top, left top, from(#0053b3), to(#003b80));
background-image: linear-gradient(to left, #0053b3, #003b80);
```

from template.data.gouv.fr

Reply:
I added the colors you indicated, but I chose to copy-paste an existant gradient from within the theme so to be sure to cover every browser... It's a bit longer than  a plein `liner-gradient` but it seems to be the way TK or other devs here chose to do before me...